utils/config/nodes.hpp

```diff
@@ -98,45 +98,16 @@ class leaf_node : public detail::base_node {
 public:
     virtual ~leaf_node(void);
 
-    /// Checks whether the node has been set by the user.
-    ///
-    /// Nodes of the tree are predefined by the caller to specify the valid
-    /// types of the leaves.  Such predefinition results in the creation of
-    /// nodes within the tree, but these nodes have not yet been set.
-    /// Traversing these nodes is invalid and should result in an "unknown key"
-    /// error.
-    ///
-    /// \return True if a value has been set in the node.
     virtual bool is_set(void) const = 0;
 
     base_node* combine(const detail::tree_key&, const base_node*) const;
 
-    /// Pushes the node's value onto the Lua stack.
-    ///
-    /// \param state The Lua state onto which to push the value.
-    virtual void push_lua(lutok::state& state) const = 0;
+    virtual void push_lua(lutok::state&) const = 0;
 
```

Comment:
nit: You could now drop this blank line to glue `push_lua` and `set_lua` together given that they seem related. Same for the two string functions below.

Reply:
Done.